XcodeServerSDK/Utils/Logging.swift

```diff
@@ -50,7 +50,7 @@ public class ConsoleLogger: Logger {
     }
     
     public func log(message: String) {
-        println(message)
+        print(message)
```

Comment:
There no more `println()` it was changed to `print()` but if we want a new line it's `print(value, appendNewline)`